NewParserTest/Node/ComparisonOperationNode.cs

```diff
@@ -0,0 +1,23 @@
+using NewPirateLexer.Tokens;
+using NewParserTest.Node.Interfaces;
+
+namespace NewParserTest.Node;
+
+public class ComparisonOperationNode : INode, IOperationNode
+{
+    public INode Left { get; set; }
```

Comment:
the properties can be privates?

Reply:
Properties are referenced in the interface